src/main/groovy/ai/digital/integration/server/tasks/worker/SyncServerPluginsWithWorkerTask.groovy

```diff
@@ -0,0 +1,69 @@
+package ai.digital.integration.server.tasks.worker
+
+import ai.digital.integration.server.domain.Worker
+import ai.digital.integration.server.util.ProcessUtil
+import ai.digital.integration.server.util.ServerUtil
+import ai.digital.integration.server.util.WorkerUtil
+import org.apache.commons.io.FileUtils
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+import java.nio.file.Paths
+
+import static ai.digital.integration.server.constant.PluginConstant.PLUGIN_GROUP
+
+class SyncServerPluginsWithWorkerTask extends DefaultTask {
+    static NAME = "syncServerPluginsWithWorker"
+
+    def dependencies = [
+            DownloadAndExtractWorkerDistTask.NAME
+    ]
+
+    SyncServerPluginsWithWorkerTask() {
+        this.configure { ->
+            group = PLUGIN_GROUP
+            dependsOn(dependencies)
+        }
+    }
+
+    @TaskAction
+    def copyToWorkers() {
+        WorkerUtil.getWorkers(project).forEach { Worker worker ->
+            copyServerDirToWorkerDir(worker)
+        }
+    }
+
+    void copyServerDirToWorkerDir(Worker worker) {
+        if (!WorkerUtil.isExternalRuntimeWorker(worker, project)) {
+            def sourceDir = ServerUtil.getServerWorkingDir(project)
+            def destinationDir = Paths.get(WorkerUtil.getWorkerWorkingDir(worker, project)).toFile()
+            destinationDir.setExecutable(true)
+            ProcessUtil.chMod(project, "755", "${destinationDir.getAbsolutePath().toString()}")
+
+            project.logger.lifecycle("Copy Worker ${worker.name} runtime from ${sourceDir} to ${destinationDir}")
+
+            // delete plugins from zip
+            FileUtils.deleteDirectory(Paths.get(WorkerUtil.getWorkerWorkingDir(worker, project), "plugins").toFile())
+
+            [
+                    "hotfix",
+                    "plugins"
+            ].forEach { String dir ->
+                FileUtils.copyDirectory(
+                        Paths.get(ServerUtil.getServerWorkingDir(project), dir).toFile(),
+                        Paths.get(WorkerUtil.getWorkerWorkingDir(worker, project), dir).toFile()
```

Comment:
Usually by convention in many languages `context` comes first. In our case it is `project`.
Let's switch parameters.

Reply:
Plus we can have such logic inside of WorkerUtil. 
For this place this code looks quite low level, same for license below

Reply:
Moved the project to the first place in arguments.

For the second comment, I changed that part and moved code to FileUtil, and now reuse that for copying files/dirs